waku/v2/protocol/filter/client.go

```diff
@@ -457,14 +459,14 @@ func (wf *WakuFilterLightNode) Ping(ctx context.Context, peerID peer.ID, opts ..
 		peerID)
 }
 
-func (wf *WakuFilterLightNode) IsSubscriptionAlive(ctx context.Context, subscription *subscription.SubscriptionDetails) error {
+func (wf *WakuFilterLightNode) IsSubscriptionAlive(ctx context.Context, subscription *subscription.SubscriptionDetails) bool {
```

Comment:
Hmm, I am not sure if this API should be kept at all...i am thinking this can be removed because now subscription liveness is anyways checked and handled already.

Will wait for @richard-ramos comments as well if i can remove this method.

Reply:
I don't think it will be required in status-go anymore, because it was only used to do pings and take action to resubscribe. But now with resubscribe being managed at API layer, status-go will not require this function anymore.
cc @vitvly

Reply:
Yes, correct,